src/views/product-downloads-view/server.ts

```diff
@@ -60,6 +60,9 @@ async function generateStaticProps(
    */
   return {
     props: stripUndefinedProperties({
+      metadata: {
+        title: 'Install',
+      },
```

Comment:
📌 Do we need a `TODO` here and in `src/views/product-tutorials-view/server.ts` to specify `description`?

Reply:
I think it's adequately captured by this ticket, so I don't feel too inclined to add a TODO here: https://app.asana.com/0/1202114367927919/1202293883440482/f